src/bidiMapper/domains/events/EventManager.ts

```diff
@@ -78,9 +78,8 @@ export class EventManager implements IEventManager {
   /**
    * Maps event name to a desired buffer length.
    */
-  static readonly #eventBufferLength: Map<string, number> = new Map([
-    ['log.entryAdded', 100],
-  ]);
+  #eventBufferLength = new Map<string, number>([['log.entryAdded', 100]]);
```

Comment:
I think it should be a const and `Readonly<Map<string, number>>`

Reply:
Done